docs/source/tutorials/token_classification.ipynb

```diff
@@ -0,0 +1,757 @@
+      "labeled as class `O` but predicted to actually be class `LOC` 3 times\n",
+      "labeled as class `MISC` but predicted to actually be class `ORG` 1 times\n",
+      "\n",
+      "Token 'Philadelphia' is potentially mislabeled 15 times throughout the dataset\n",
+      "---------------------------------------------------------------------------------------\n",
+      "labeled as class `ORG` but predicted to actually be class `LOC` 14 times\n",
+      "labeled as class `LOC` but predicted to actually be class `ORG` 1 times\n",
+      "\n",
+      "Token 'Usda' is potentially mislabeled 13 times throughout the dataset\n",
+      "---------------------------------------------------------------------------------------\n",
+      "labeled as class `ORG` but predicted to actually be class `LOC` 7 times\n",
+      "labeled as class `ORG` but predicted to actually be class `PER` 5 times\n",
+      "labeled as class `ORG` but predicted to actually be class `MISC` 1 times\n",
+      "\n",
+      "Token 'York' is potentially mislabeled 12 times throughout the dataset\n",
+      "---------------------------------------------------------------------------------------\n",
+      "labeled as class `ORG` but predicted to actually be class `LOC` 11 times\n",
+      "labeled as class `LOC` but predicted to actually be class `ORG` 1 times\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "info = common_label_issues(issues, given_words, \n",
+    "                           labels=labels, \n",
+    "                           pred_probs=pred_probs, \n",
+    "                           class_names=entities, \n",
+    "                           exclude=[(0, 1), (1, 0)]) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "01614ffd",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "             token given_label predicted_label  num_label_issues\n",
+      "0                /           O             LOC                36\n",
+      "1          Chicago         ORG             LOC                22\n",
+      "2            Press           O             ORG                20\n",
+      "3           Digest           O             ORG                20\n",
+      "4     Philadelphia         ORG             LOC                14\n",
+      "...            ...         ...             ...               ...\n",
+      "1222      Chechnya         LOC            MISC                 1\n",
+      "1223      Counties         ORG            MISC                 1\n",
+      "1224           Red        MISC             ORG                 1\n",
+      "1225           Ods         ORG            MISC                 1\n",
+      "1226       Indians        MISC             ORG                 1\n",
+      "\n",
+      "[1227 rows x 4 columns]\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(info)"
+   ]
+  },
```

Comment:
Don't think we need this cell. Can just say (in markdown text) that:  

The above printed information is also stored as a DataFrame `info`.